doc/classes/Node.xml

```diff
@@ -331,7 +324,7 @@
 				Returns the time elapsed since the last physics-bound frame (see [method _physics_process]). This is always a constant value in physics processing unless the frames per second is changed via [member Engine.iterations_per_second].
 			</description>
 		</method>
-		<method name="get_position_in_parent" qualifiers="const">
+		<method name="get_index" qualifiers="const">
```

Comment:
`get_index` is no longer sorted properly in the documentation due to how this change was made.